apps/mail/app/(routes)/mail/[folder]/[threadId]/page.tsx

```
@@ -1,13 +1,65 @@
+  const { pageToken, q, max, labelIds } = await searchParams;
+  const threadsResponse = await getMails({
+    folder,
+    q,
+    max,
+    pageToken,
+    labelIds,
+  });
+
+  let threadMessages: ParsedMessage[] = [];
+  if (threadId) threadMessages = (await getMail({ id: threadId })) ?? [];
+  if (threadMessages && threadMessages.some((e) => e.unread)) void markAsRead({ ids: [threadId] });
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)